types.go

```diff
@@ -538,7 +594,7 @@ func readPLPType(ti *typeInfo, r *tdsBuffer) (res interface{}) {
 }
 
 func writePLPType(w io.Writer, ti typeInfo, buf []byte) (err error) {
-	if err = binary.Write(w, binary.LittleEndian, uint64(len(buf))); err != nil {
+	if err = binary.Write(w, binary.LittleEndian, uint64(UNKNOWN_PLP_LEN)); err != nil {
```

Comment:
I think when the size is known it should be reported as is, not as unknown.  Why do you report it as unknown?